pandas/core/window.py

```diff
@@ -1028,19 +1028,113 @@ def _get_cov(X, Y):
                                    _get_cov, pairwise=bool(pairwise))
 
     _shared_docs['corr'] = dedent("""
-    %(name)s sample correlation
+    Calculate %(name)s correlation.
+
+    This function uses Pearson's definition of correlation 
+    (https://en.wikipedia.org/wiki/Pearson_correlation_coefficient).
 
     Parameters
     ----------
     other : Series, DataFrame, or ndarray, optional
-        if not supplied then will default to self and produce pairwise output
+        If not supplied then will default to self.
     pairwise : bool, default None
-        If False then only matching columns between self and other will be
-        used and the output will be a DataFrame.
-        If True then all pairwise combinations will be calculated and the
-        output will be a MultiIndex DataFrame in the case of DataFrame inputs.
-        In the case of missing elements, only complete pairwise observations
-        will be used.""")
+        Calculate pairwise combinations of columns within a
+        DataFrame. If `other` is not specified, defaults to `True`,
+        otherwise defaults to `False`. Not relevant for :class:`~pandas.Series`.
+        See notes.
+    **kwargs
```

Comment:
IIRC we remove this. @TomAugspurger

Reply:
This line is fine.

For the explanation, you can put "For compatibility with other %(name)s methods. Not used."